bloom-housing/bloom
# 1723/remove business logic from HouseholdSizeField component.

# Pull Request Template

## Issue

Addresses #1723


## Description

Removes business logic from HouseholdSizeField component.

## Type of change


## How Can This Be Tested/Reviewed?

Check out the new story! We don't have household validation turned on in Alameda or any local listings. Might be a component we deprecate eventually after seeing what other jurisdictions want. This is an "invisible" form field that validates household size. I really feel like eventually this should just be a component we toggle on and off at the app level.

## Checklist:
